src/client.cpp

```diff
@@ -688,4 +688,5 @@ void PauseStream(bool _UNUSED(bPaused)) {}
 bool SeekTime(int, bool, double*) { return false; }
 void SetSpeed(int) {};
 bool IsTimeshifting(void) { return false; }
+bool IsRealTimeStream(void) { return true; }
```

Comment:
As explained in the dvbviewer subforum (you probably haven't read this, but @Uukrull might have), I'll adapt the PVR in the next days. `IsTimeshifting` returning false is definitely wrong.

Btw, when is feature freeze for Krypton?